### PR TITLE
add pod info when failing to add pod to unschedulableQ

### PR DESCRIPTION
**Pull request by cwdsuzhou**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

add pod info when failing to add pod to unschedulableQ

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Review by wgliang — APPROVED** [review]
/ok-to-test
/lgtm

**Comment by alculquicondor** [comment]
Cc @liu-cong

File: pkg/scheduler/internal/queue/scheduling_queue.go [hunk 1]
```diff
@@ -329,7 +329,7 @@ func (p *PriorityQueue) AddUnschedulableIfNotPresent(pInfo *framework.PodInfo, p
 	defer p.lock.Unlock()
 	pod := pInfo.Pod
 	if p.unschedulableQ.get(pod) != nil {
-		return fmt.Errorf("pod is already present in unschedulableQ")
+		return fmt.Errorf("pod: %v/%v is already present in unschedulableQ", pod.Namespace, pod.Name)
```
Comment (liu-cong) [inline pkg/scheduler/internal/queue/scheduling_queue.go]: Can you change `unschedulableQ` to `unschedulable queue`?
Can you also make the same change for lines(338 and 341) below for active and backoff queues?

Thanks

Comment (cwdsuzhou) [reply pkg/scheduler/internal/queue/scheduling_queue.go]: done

**Comment by cwdsuzhou** [comment]
ping @wgliang  for approval

File: pkg/scheduler/internal/queue/scheduling_queue.go [hunk 2]
```diff
@@ -329,16 +329,16 @@ func (p *PriorityQueue) AddUnschedulableIfNotPresent(pInfo *framework.PodInfo, p
...
 	if p.unschedulableQ.get(pod) != nil {
-		return fmt.Errorf("pod is already present in unschedulableQ")
+		return fmt.Errorf("pod: %v/%v is already present in unschedulable queue", pod.Namespace, pod.Name)
 	}
 
 	// Refresh the timestamp since the pod is re-added.
 	pInfo.Timestamp = p.clock.Now()
 	if _, exists, _ := p.activeQ.Get(pInfo); exists {
-		return fmt.Errorf("pod is already present in the activeQ")
+		return fmt.Errorf("pod is already present in the active queue")
```
Comment (liu-cong) [inline pkg/scheduler/internal/queue/scheduling_queue.go]: can you also print the pod namespace and name in the error message? Same for the backoff queue line below

File: pkg/scheduler/internal/queue/scheduling_queue.go [hunk 3]
```diff
@@ -329,16 +329,16 @@ func (p *PriorityQueue) AddUnschedulableIfNotPresent(pInfo *framework.PodInfo, p
 	defer p.lock.Unlock()
 	pod := pInfo.Pod
 	if p.unschedulableQ.get(pod) != nil {
-		return fmt.Errorf("pod is already present in unschedulableQ")
+		return fmt.Errorf("pod: %v/%v is already present in unschedulable queue", pod.Namespace, pod.Name)
```
Comment (liu-cong) [inline pkg/scheduler/internal/queue/scheduling_queue.go]: nit: seems that you can use the `nsNameForPod` helper method.

File: pkg/scheduler/internal/queue/scheduling_queue.go (hunk 2 above)
Comment (cwdsuzhou) [reply pkg/scheduler/internal/queue/scheduling_queue.go]: seems my latest commit did not send successfully

Updated it.

File: pkg/scheduler/internal/queue/scheduling_queue.go (hunk 3 above)
Comment (cwdsuzhou) [reply pkg/scheduler/internal/queue/scheduling_queue.go]: done

**Comment by cwdsuzhou** [comment]
/assign @bsalamat

ping @bsalamat for approval

**Review by bsalamat — COMMENTED** [review]
/approve

**Pull request merged**